src/ISDisplay.cpp

```diff
@@ -225,19 +225,41 @@ void cInertialSenseDisplay::GoToColumnAndRow(int x, int y)
 
 string cInertialSenseDisplay::Hello()
 {
-	return "$ Inertial Sense.  Press CTRL-C to terminate.\n";
+	return "$ Inertial Sense.  CTRL-C to terminate.\n";
 }
 
 string cInertialSenseDisplay::Connected()
 {
-	return string("$ Inertial Sense.  Connected.  Press CTRL-C to terminate.  Rx ") + std::to_string(m_rxCount) + "\n";
+	if (m_startMs==0)
+	{
+		m_startMs = current_timeMs();
+	}
+
+	// cltool runtime
+	double runtime = 0.001 * (current_timeMs() - m_startMs);
+
+	std::ostringstream stream;
+
+	stream << "$ Inertial Sense.  Connected.  CTRL-C to terminate.  ";
```

Comment:
Its not a big deal, and it was this way before your work, but we should try and consolidate these messages/string.  Would a `generateHeader(bool connected)` make more sense, and then have the resulting string be a concatenation of these fields, if we're connected (or not)?

Reply:
Done